src/phrases/phrases-en.js

```diff
@@ -587,6 +587,14 @@ const phrases = {
       add: 'Choose meta image',
       remove: 'Remove meta image',
     },
+    availability: {
+      everyone: 'Everyone',
+      teacher: 'Teachers',
+      student: 'Students',
+      label: 'Availability',
+      labelChild: 'Choose availability:',
+      description: 'Articles availability',
```

Comment:
jeg er usikker men kanskje:
```suggestion
      labelChild: 'Choose the availability:',
      description: 'Availability of the article',
```